client/landing/stepper/declarative-flow/flows/onboarding/onboarding.ts

```diff
@@ -94,10 +101,15 @@ const onboarding: Flow = {
 			[ isGoalsAtFrontExperiment, initialLoggedOut, initialGoals, isLoading ]
 		);
 	},
-	useSteps() {
-		// We have already checked the value has loaded in useAssertConditions
-		const [ , isGoalsAtFrontExperiment ] = useGoalsFirstExperiment();
```

Comment:
This is safe to remove because `useGoalsFirstExperiment` is currently hardcoded to `false`.

I'm cleaning up more uses of this hook in https://github.com/Automattic/wp-calypso/pull/102577. I just didn't want to do all the clean up in this PR.

Reply:
> async initialize()

Thanks so much for using the new Stepper tools!

> This is safe to remove because useGoalsFirstExperiment is currently hardcoded to false.

Sounds good!

> I'm cleaning up more uses of this hook in #102577.

Thanks for spliting it in another PR! We're also doing [a cleanup work here](https://github.com/Automattic/wp-calypso/pull/101921#issuecomment-2782735281).